django_project/feti/static/feti/css/landing-page.css

```diff
@@ -546,7 +546,7 @@
 .result-title img {
     width: 54px;
     height: auto;
-    margin-right: 20px;
+    margin: 10px 5px 5px 5px;
```

Comment:
Yes, we don't need additional margin space. 
And this is what it looks like in your PR:
![ford_2](https://cloud.githubusercontent.com/assets/1979569/25113971/60bbddce-2425-11e7-8a28-2471cd88620b.png)

Still not centered vertically.